geode-redis/README.md

```diff
@@ -159,54 +159,54 @@ start server \
+| HSCAN                 | SCARD                                                 | BZPOPMAX                      	|
+| HSET                  | SDIFF                                                 | BZPOPMIN                      	|
+| HSETNX                | SDIFFSTORE                                            | CLIENT CACHING                	|
+| HSTRLEN               | SELECT                                                | CLIENT GETNAME                	|
+| HLEN                  | SETBIT                                                | CLIENT GETREDIR               	|
+| HVALS                 | SETEX                                                	| CLIENT ID                     
+| INCRBY                | SETRANGE                                              | CLIENT KILL                   	|
+| INCRBYFLOAT           | SHUTDOWN                                              | CLIENT LIST                   	|
+| KEYS                  | SINTER                                                | CLIENT PAUSE                  	|
+| MGET                  | SINTERSTORE                                           | CLIENT REPLY                  	|
+| PERSIST               | SISMEMBER                                             | CLIENT SETNAME                	|
+| PEXPIRE               | SLOWLOG                                               | CLIENT TRACKING               	|
+| PEXPIREAT             | SMOVE                                                 | CLIENT UNBLOCK                	|
+| PING                  | SPOP                                                  | CLUSTER ADDSLOTS              	|
+| PSUBSCRIBE            | SRANDMEMBER                                           | CLUSTER BUMPEPOCH             
+| PTTL                  | SSCAN                                                 | CLUSTER COUNT-FAILURE-REPORTS 	|
+| PUBLISH               | SUNION                                                | CLUSTER COUNTKEYSINSLOT       	|
+| PUNSUBSCRIBE          | SUNIONSTORE                                           | CLUSTER DELSLOTS     	            |
+| QUIT                  | TIME                                                  | CLUSTER FAILOVER              	|
+| RENAME                | UNLINK [1]                                            | CLUSTER FLUSHSLOTS                |
+| SADD                                                                          | CLUSTER FORGET                    |
+| SET                                                                           | CLUSTER GETKEYSINSLOT             |
+| SETNX              	                                         	            | CLUSTER INFO                  	|
+| SMEMBERS                                                       	            | CLUSTER KEYSLOT                   |
+| SREM                                                                          | CLUSTER MEET                      |
```

Comment:
Because some of the `|` characters were not added back here, the `CLUSTER FORGET` through `CLUSTER MEET` commands are in the Unsupported column.

Reply:
added back.  thanks